packages/e2e/cypress/integration/common/index.tsx

```diff
@@ -15,8 +15,8 @@ And('I type a valid password {string}', (password: string) => {
   cy.findByLabelText(/password/i).type(Cypress.env(password));
 });
 
-And('I click the {string} button', (name: string) => {
-  cy.findByRole('button', { name }).click();
+When('I click the {string} button', (name: string) => {
```

Comment:
This step definition can be used by any test, so I just updated this one to be case insensitive so all tests can use this.